File: floorball-view/src/components/Crashlog.js
//import React from 'react';

const CrashLog = (err, file, line) => {
  const data= {
    "Date": new Date(),
    "DebugMode": 0,
    "Info": err,
    "File": file,
    "Line": line
  }
  console.log('Crash körs')

  localStorage.setItem('myCat', JSON.stringify(data));
}

export default CrashLog;

File: floorball/Models/CrashLog.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace floorball.Models
{
    public class CrashLog
    {
        public int Id { get; set; }
        public string Date { get; set; }
        public int DebugMode { get; set; }
        public string Info { get; set; }
        public string File { get; set; }
        public int Line { get; set; }

    }
}
